nixos/modules/services/networking/atticd.nix

```diff
@@ -0,0 +1,241 @@
+        ];
+
+        services.atticd.settings = {
+          chunking = lib.mkDefault {
+            nar-size-threshold = 65536;
+            min-size = 16384; # 16 KiB
+            avg-size = 65536; # 64 KiB
+            max-size = 262144; # 256 KiB
+          };
+
+          database.url = lib.mkDefault "sqlite:///var/lib/atticd/server.db?mode=rwc";
+
+          # "storage" is internally tagged
+          # if the user sets something the whole thing must be replaced
+          storage = lib.mkDefault {
+            type = "local";
+            path = "/var/lib/atticd/storage";
+          };
+        };
+
+        systemd.services.atticd = {
+          wantedBy = [ "multi-user.target" ];
+          after =
+            [ "network.target" ]
+            ++ lib.optionals hasLocalPostgresDB [
+              "postgresql.service"
+              "nss-lookup.target"
+            ];
+          serviceConfig = {
+            ExecStart = "${lib.getExe cfg.package} -f ${checkedConfigFile} --mode ${cfg.mode}";
+            EnvironmentFile = cfg.environmentFile;
+            StateDirectory = "atticd"; # for usage with local storage and sqlite
+            DynamicUser = true;
+            User = cfg.user;
+            Group = cfg.group;
+            Restart = "on-failure";
+            RestartSec = 10;
+
+            CapabilityBoundingSet = [ "" ];
+            DeviceAllow = "";
+            DevicePolicy = "closed";
+            LockPersonality = true;
+            MemoryDenyWriteExecute = true;
+            NoNewPrivileges = true;
+            PrivateDevices = true;
+            PrivateTmp = true;
+            PrivateUsers = true;
+            ProcSubset = "pid";
+            ProtectClock = true;
+            ProtectControlGroups = true;
+            ProtectHome = true;
+            ProtectHostname = true;
+            ProtectKernelLogs = true;
+            ProtectKernelModules = true;
+            ProtectKernelTunables = true;
+            ProtectProc = "invisible";
+            ProtectSystem = "strict";
+            ReadWritePaths =
+              let
+                path = cfg.settings.storage.path;
+                isDefaultStateDirectory = path == "/var/lib/atticd" || lib.hasPrefix "/var/lib/atticd/" path;
+              in
+              lib.optionals (cfg.settings.storage.type or "" == "local" && !isDefaultStateDirectory) [ path ];
+            RemoveIPC = true;
+            RestrictAddressFamilies = [
+              "AF_INET"
+              "AF_INET6"
+              "AF_UNIX"
+            ];
+            RestrictNamespaces = true;
+            RestrictRealtime = true;
+            RestrictSUIDSGID = true;
+            SystemCallArchitectures = "native";
+            SystemCallFilter = [
+              "@system-service"
+              "~@resources"
+              "~@privileged"
+            ];
+            UMask = "0077";
+          };
+        };
+
+        environment.systemPackages = [
+          atticadmWrapper
+          pkgs.attic-client
```

Comment:
is it really intended to install the client also on the server? i have the client just installed on those machines that act as builders and not on every machine.
Also upstream module does not include a client